kolibri/plugins/coach/assets/src/views/exams-page/delete-exam-modal.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+
+  <core-modal :title="$tr('deleteExam')" @cancel="close">
+    <p>
+      <span v-html="$trHtml('areYouSure', { examTitle })"></span>
+      {{ $tr('youWillLose') }}
+    </p>
+    <icon-button :text="$tr('cancel')" @click="close"/>
+    <icon-button :text="$tr('delete')" :primary="true" @click="deleteExam"/>
+  </core-modal>
+
+</template>
+
+
+<script>
+
+  const examActions = require('../../state/actions/exam');
+
+  module.exports = {
+    $trNameSpace: 'deleteExamModal',
+    $trs: {
+      deleteExam: 'Delete exam',
+      areYouSure: 'Are you sure you want to delete <strong>{ examTitle }</strong>?',
+      youWillLose: 'You will lose all data for this exam.',
```

Comment:
> This is a dialog with multiple sentences, so It looks appropriate.

That seems correct.

It might make sense to combine `areYouSure` and `youWillLose` into a single confirmation string so that translator (and devs) can see that they are two sentences in the same paragraph.